src/Mgate/PersonneBundle/Resources/translations/person.fr.yml

```diff
@@ -0,0 +1,155 @@
+generic:
```

Comment:
That doesn't make sense though, it's as for the code in french, it doesn't make sense when the language doesn't support well the language. Either use generic translations keys (so English is the preferred choice) or don't use translations at all.